resources/helm/v2.1/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -94,10 +92,6 @@ spec:
           - name: webhook
             containerPort: 15017
             protocol: TCP
-          - containerPort: 15053
-            protocol: TCP
-          - containerPort: 8188
```

Comment:
This is for discovery, although this change will need to be made in maistra/istio

Reply:
This is back.